### PR TITLE
UAT Fix/CLOUD-339

### PR DESCRIPTION
**Pull request by DavidCramer**
on upgrade from v1, set auto upload to off. Keep default (new install at v2) to on.

### CODE REVIEW

File: cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/class-connect.php [hunk 1]
```diff
@@ -338,7 +338,7 @@ public function get_config() {
 				$sync = get_option( 'cloudinary_sync_media' );
 				if ( empty( $sync ) ) {
 					$sync = array(
-						'auto_sync'         => '',
+						'auto_sync'         => 'off',
```
Comment (pereirinha) [inline cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/class-connect.php:341]: @DavidCramer 

Won't this be overridden by line 345?

**Comment by DavidCramer** [comment]
Please do. Let's chat tomorrow on it!

**Comment by dugajean** [comment]
How's this coming along?

**Comment by pereirinha** [comment]
Closing the PR as this was fixed on 2.1.1

**Pull request closed without merging**